content/en/docs/ops/best-practices/security/index.md

```diff
@@ -90,6 +90,80 @@ spec:
+
+The enforcement point of authorization policies is the Envoy proxy which is different from the actual resource access point that is
+usually in the backend application. A policy mismatch happens when the Envoy proxy and the backend application interprets the request
+differently.
+
+The mismatch leads to either unexpected rejection or policy bypass. The latter is usually a security incident that needs to be
+fixed immediately, and it's also why we need path normalization in the authorization policy.
+
+For example, you have an authorization policy to reject requests with path `/data/secret`. A request with path `/data//secret` will
+not be rejected because it does not match the path defined in the authorization policy due to the extra forward slash `/` in the path.
+
+The request goes through and later the backend application returns the same response that it returns for the path `/data/secret`
+because the backend application normalizes the path `/data//secret` to `/data/secret` as it considers the double forward slashes
+`//` equivalent to a single forward slash `/`.
+
+In this example, the policy enforcement point (Envoy proxy) had a different understanding of the path than to the resource access
+point (backend application). The different understanding caused the mismatch and subsequently the bypass of the authorization policy.
+
+This becomes a complicated problem because of the following factors:
+
+* Lacking of a clear standard of the normalization.
+
+* Backends and frameworks in different layers have their own special normalization.
+
+* Applications can even have arbitrary normalizations for its own use cases.
+
+Istio authorization policy implements built-in support of various basic normalization options to help you to better address
+the problem:
+
+* Read the [Guideline on configuring the path normalization option](/docs/ops/best-practices/security/#guideline-on-configuring-the-path-normalization-option)
+  to understand which normalization options you may want to use.
+
+* Read the [Customize your system on path normalization](/docs/ops/best-practices/security/#customize-your-system-on-path-normalization) to
+  understand the detail of each normalization option.
+
+* Read the [Mitigation for unsupported normalization](/docs/ops/best-practices/security/#mitigation-for-unsupported-normalization) for
+  alternative solutions in case you need any unsupported normalization options.
+
+### Guideline on configuring the path normalization option
+
+#### Case 1: You do not need normalization at all
+
+Before diving into the details of configuring normalization, you should first make sure whether any of the normalizations are needed.
+
+You do not need the normalization if you do not use authorization policies or if your authorization policies do not
+use any `path` fields.
+
+You may not need the normalization if all your authorization policies follow the safer authorization pattern which, in the worst
```

Comment:
what is the "safer authorization pattern"? We should link it from here

Reply:
added link